### ISSUE TITLE
Mobile First Design (Mobile Global Styles)

### ISSUE THREAD

**Issue by ViktorIwan** [soflyy/oxygen-bugs-and-features#126]
At the moment, i'm doing for Mobile First Design, but your Global Style is fixed on desktop, i think it would be great if your Global Style also support for responsiveness, so either people who use desktop first or mobile first approach can get the same confinient

Please consider

**Comment by Ikpixel** [comment]
Yes please give us the options (toggle) to choose if we want to use mobile first or desktop first.

**Comment by adrien-robert** [comment]
+1

**Comment by LauGau** [comment]
+1 here

**Comment by CasualProtagonist** [comment]
++

**Comment by fedeescalada** [comment]
+1

**Comment by Anibaaal** [comment]
Much needed for optimizing loading speed for mobile. And if I can suggest, this would go along well with an option to save media queries in a separate stylesheet.

**Comment by aleklisefski** [comment]
+1 please!

**Comment by AztecInternet** [comment]
+1 Deffo a great option if you decide to implement it. You could simply select "Mobile First" in a pair of radio buttons in Oxygen Settings and the media queries flip between @min-width and @max-width, something like that. Or I guess just a "mobile First" tick (check) box.

**Comment by flick36** [comment]
+1

**Comment by AlkoKod** [comment]
+1

**Comment by AztecInternet** [comment]
+1

**Comment by Astwert** [comment]
+1

**Comment by kalaio** [comment]
+1

**Comment by Astwert** [comment]
+1

**Comment by adrianmarmy** [comment]
Also I think it would be consistent to Oxygen forcing people to actually use best practice of real web design just like it uses the real terms like div, h1, flexbox and so on.
I'm not a pro but I'm under the impression that front end development today works mobile first which to me would actually make total sense since what works in mobile for sure works on a desktop while what works on desktop usually never works on mobile. 

So at least a toggle would be awesome. I would totally switch to mobile first. :-)

**Comment by YammYcoding** [comment]
+1

**Comment by kcxmedia** [comment]
+1 The solution for this could be as simple as a toggle on the responsive panel to change each breakpoint from Less than *998 (or whatever) to More than *998 (or whatever).

**Comment by jvorld** [comment]
+1

**Comment by derSebastian** [comment]
Yes to a mobile first design “mode”

**Comment by studioavanti** [comment]
+1

**Comment by iosisl** [comment]
+1 Since it's been a while since google started mobile first it's logical to at least have that option. Right?

**Comment by kevinelgan** [comment]
+1

**Comment by schietegal** [comment]
+1

**Comment by moodworks** [comment]
Please, +1

**Comment by Pearly12** [comment]
+1

**Comment by arevesz** [comment]
+1,000,001

**Comment by websitesbyjeremy** [comment]
+1 please

**Comment by hoadotrang** [comment]
+111 Please

**Comment by stephde123** [comment]
+1

**Comment by arabbittpro** [comment]
+1 - More flexibility in media queries, please. Ideally, in global styles, we'd be able to define any px value in a field (such as we can now) but also choose whether or not it's a min or max value via the field's respective dropdown (maybe even having the option for a range, meaning between two breakpoints). Then, however it's defined via these global styles would appear in the breakpoint list on the left.

**Comment by braeu71** [comment]
+1

**Comment by christianfynbo** [comment]
In a world of the mobile first index – for a builder that boasts on how much Google will love you – it is really odd that this was not implemented years ago. Mobile indexing was rolled out officially in 2016 and made default for all new sites mid 2019.

**Comment by reachingmike** [comment]
+1

**Comment by vicento** [comment]
✅ we should be able to design websites  with a **mobile first perspective now in Oxygen**, Oxygen team must consider that

**Comment by czaczawik** [comment]
+1

**Comment by demetre19** [comment]
+1

**Comment by KonradDeskiewicz** [comment]
Geeez... more than 2 years from posting and we're still stuck with this one. C'mon guys... I understand that development is slow because the wrong framework was picked at the start for the Oxygen engine but still... this is like a BASIC thing to do to make our life a tad bit easier. Now I have all defaults removed from those fields and a bunch of classes created that can be responsive. The only thing is I need to remember to add them EVERY TIME :(

**Comment by NicolaModugno** [comment]
+1

**Comment by lucilecollignon** [comment]
+1

**Comment by LauraHamburg** [comment]
+10

**Comment by uhlhosting** [comment]
+1

**Comment by jevgen** [comment]
+1
